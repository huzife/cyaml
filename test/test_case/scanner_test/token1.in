---
- 123456
- test
-
  string: >-
    this
    is
    a
    test
    string
