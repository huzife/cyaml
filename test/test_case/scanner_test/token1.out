(STREAM_START)#
(DOC_START)#
(BLOCK_SEQ_START)#
(BLOCK_SEQ_ENTRY)#
(SCALAR, 123456)#
(BLOCK_SEQ_ENTRY)#
(SCALAR, test)#
(BLOCK_SEQ_ENTRY)#
(BLOCK_MAP_START)#
(KEY, string)#
(SCALAR, this is a test string)#
(BLOCK_MAP_END)#
(BLOCK_SEQ_END)#
(STREAM_END)#
